x/auth/mock/app.go

```diff
@@ -0,0 +1,110 @@
+package mock
```

Comment:
I think this might conflict with what's on the latest `develop`.

Reply:
++, this file shoudn't be in this PR

Reply:
I'm not sure how this got added? How do I get rid of it?